static/app/components/events/autofix/useAutofix.tsx

```diff
@@ -132,10 +129,7 @@ export const useAiAutofix = (group: GroupWithAutofix, event: Event) => {
             instruction,
           },
         });
-        // Save the run_id from the response
-        if (response.autofix?.run_id) {
-          setCurrentRunId(response.autofix.run_id);
-        }
+        setCurrentRunId(response.run_id);
```

Comment:
```suggestion
        setCurrentRunId(response.run_id ?? null);
```
tiny nit: Not all the api responses return a run id, for example the error ones, so this ensures it's still null not undefined